bump.config.yml

```diff
@@ -0,0 +1,5 @@
+files:
+  - filePath: './eslint/package.json'
+    patterns:
+      - '"name"\s*:\s*"\@exadel\/eslint-plugin-esl"[^{}]*"version"\s*:\s*"(\d+\.\d+\.\d+)"'
+      - '"@exadel\/esl\": \"\^(\d+\.\d+\.\d+)\"'
```

Comment:
There is no script to call bump.js in the bounds of the root package. Moreover, now, it should not be called in the scope of the root package.

The version control is currently handled by npm built-in script flow.
We need to update the flow of version change of the exact package using the version provided by the npm arguments (but not the root package)
Please consider registering a post (version or postversion) hook inside of eslint module to manage related versions.